clio-client/src/main/scala/org/broadinstitute/clio/client/dispatch/CommandDispatch.scala

```diff
@@ -22,6 +22,7 @@ class CommandDispatch(val webClient: ClioWebClient, val ioUtil: IoUtil)
       case addCommand: AddCommand[_]       => new AddExecutor(addCommand)
       case moveCommand: MoveCommand[_]     => new MoveExecutor(moveCommand)
       case deleteCommand: DeleteCommand[_] => new DeleteExecutor(deleteCommand)
+      case deliverCommand: DeliverWgsCram  => new DeliverExecutor(deliverCommand)
```

Comment:
Are we ever going to make `deliver` do more than just WgsCrams?

Reply:
Unsure. For now, I'd rather start specific and generalize if needed than make it sound general but actually only deal with crams.